talk/owt/sdk/p2p/p2pclient.cc

```diff
@@ -393,6 +393,13 @@ void P2PClient::OnStopped(const std::string& remote_id) {
     removed_pc_ = pc_channels_[remote_id];
     pc_channels_.erase(remote_id);
   }
+  // remove from allowed ids to prevent memory leaks.
+  {
+    const std::lock_guard<std::mutex> lock(remote_ids_mutex_);
+    allowed_remote_ids_.erase(
+        std::remove(allowed_remote_ids_.begin(), allowed_remote_ids_.end(), remote_id),
```

Comment:
this will delete stuff that shouldn't be deleted. it erases stuff in the vector from the matched ID to the end